ingestion/functions/parsing/ch_zurich/zurich.py

```diff
@@ -0,0 +1,101 @@
+import os
+import sys
+from datetime import date, datetime
+import csv
+
+# Layer code, like parsing_lib, is added to the path by AWS.
+# To test locally (e.g. via pytest), we have to modify sys.path.
+# pylint: disable=import-error
+if ('lambda' not in sys.argv[0]):
+    sys.path.append(
+        os.path.join(
+            os.path.dirname(os.path.dirname(os.path.abspath(__file__))),
+            'common/python'))
+import parsing_lib
+
+# Fixed location, all cases are for the canton of Zurich in Switzerland.
+_LOCATION = {
+    "country": "Switzerland",
+    "administrativeAreaLevel1": "Zurich",
+    "geoResolution": "Admin1",
+    "name": "Zurich canton",
+    "geometry": {
+        "longitude": "8.651071",
+        "latitude": "47.42568"
+    }
+}
+
+_DATE_INDEX = 0
+_AGE_INDEX = 2
+_GENDER_INDEX = 3
+_CONFIRMED_INDEX = 4
+_SOURCE_INDEX = 6
+
+def convert_date(raw_date):
+    """
+    Convert raw date field into a value interpretable by the dataserver.
+
+    The date is listed in YYYY-mm-dd format, but the data server API will
```

Comment:
This doesn't need to be converted -- YYYY-mm-dd is correctly interpreted by JS's Date (on the data server). You can verify this in console. The only ambiguous case, AFAICT, is dd-mm-YYYY.

Reply:
ohhh, interesting I didn't know.
Okay then less code is better.
Removed.